### PR TITLE
WFLY-9294 Add security configuration for Infinispan remote cache store

### PR DESCRIPTION
**Pull request by rhusar**
https://issues.jboss.org/browse/WFLY-9294
https://issues.jboss.org/browse/EAP7-942

### CODE REVIEW

**Comment by rhusar** [comment]
FYI @mvinkler -- I assume you will be the QE contact for this?

**Comment by rhusar** [comment]
According to information from @mvinkler updated the QE contact to @LittleJohnII

**Comment by rhusar** [comment]
@mvinkler is the QE contact for this.

File: clustering/infinispan/WFLY-9294_Add_security_configuration_for_Infinispan_remote_cache_store.adoc [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+* WildFly/EAP
+* Infinispan Server/JDG
+
+=== Other Interested Projects
+
+N/A
+
+== Requirements
+
+=== Hard Requirements
```
[the remark this answers is not in the document]
Comment (rhusar) [reply clustering/infinispan/WFLY-9294_Add_security_configuration_for_Infinispan_remote_cache_store.adoc]: Please refer to the documentation for the change: https://github.com/wildfly/wildfly/pull/10960/files#diff-b47b6c67739c872da45b8dfe6a65e4b7

I am hesitant to put implementation details into the proposal.

**Comment by kabir** [comment]
@mvinkler I'm going to approve this one, let me know if that is wrong

File: clustering/infinispan/WFLY-9294_Add_security_configuration_for_Infinispan_remote_cache_store.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (mvinkler) [reply clustering/infinispan/WFLY-9294_Add_security_configuration_for_Infinispan_remote_cache_store.adoc]: While the documentation describes how to configure remote-cache-container and hotrod store, it doesn't mention the main goal of this RFE - to secure communication from a cache store to remote Infinispan/JDG cluster. Can you provide the configuration steps on both EAP and JDG side?

[the remark this answers is not in the document]
Comment (rhusar) [reply clustering/infinispan/WFLY-9294_Add_security_configuration_for_Infinispan_remote_cache_store.adoc]: This is described in the "Security" section of the documentation including a CLI command: https://github.com/wildfly/wildfly/pull/10960/files#diff-b47b6c67739c872da45b8dfe6a65e4b7R66

Securing Infinispan Server is in the Infinispan docs as linked from the WF docs: http://infinispan.org/docs/stable/server_guide/server_guide.html#security:hotrod_rest_encryption

Let me see if I find some scripts to help you with the setup.

**Pull request merged**